### ISSUE TITLE
bagging

### ISSUE THREAD

**Issue by ajschumacher** [ajschumacher/ajschumacher.github.io#376]
Is bagging important for neural nets? Is stochastic gradient descent giving more generalization than batch would?
